README.md

```diff
@@ -1,5 +1,10 @@
-# crocks.js
-`crocks` is a collection of popular *Algebraic Data Types (ADTs)* that are all the rage in functional programming. You have heard of things like `Maybe` and `Either` and heck maybe even `IO`, that is what these are. The main goal of `crocks` is to curate and provide not only a common interface between each type (where possible of course), but also all of the helper functions needed to hit the ground running.
+<img style="display: block; width: 360px; margin: 24px auto;" src="https://cdn.rawgit.com/rstegg/crocks/dfdee636/crocks_js.svg" />
+
+# a collection of popular *Algebraic Data Types*
+
+The Algebraic Data Types (ADT)s included in `crocks` are those which are trending in functional programming. If you've ever heard of `Maybe` or `Either`, or even `IO`, you're in the right place! The main goal of `crocks` is to provide a *curated common interface* between each type and include all the helper functions needed to get the wheel spinning.
```

Comment:
I do not understand the need for this rewrite?
Could you provide your reasoning?